### PR TITLE
Fix TrafficPimps Scraper

### PR DESCRIPTION
**Pull request by SpedNSFW**
There are quite a few changes here. CDP is required to get the cover image.

Just a couple notes regarding the URL changes:

 - The pimp.xxx, family.xxx, and petite.xxx URLs redirect to cherrypimps.com and are thus pointless.
 - The wildoncam.com URL got commented out as the site has not updated to the new layout that is present on cherrypimps.com and thus none of the selectors will work. Wild On Cam scenes _are_ present on cherrypimps.com however so if they need to be scraped, use the CherryPimps site.

### CODE REVIEW

**Comment by SpedNSFW** [comment]
I have removed the three `.xxx` sites from the scrapers list. I have left Wild On Cam there as I suspect the site will eventually be updated to the new layout at some point.

File: scrapers/trafficpimps.yml [hunk 1]
```diff
@@ -2,43 +2,30 @@ name: trafficpimps
...
       Studio:
-        Name: //div[@class="container"]/a[@class="logo"]/@title
-      Image: 
-        selector: //script[contains(.,"contentthumbs")]/text()|//img[@class="lazyload update_thumb thumbs stdimage"]/@src
-        postProcess:
-          - replace:
-              - regex: (.+)(https.+contentthumbs.+)(" width="100%)(.+)
-                with: $2
+        Name: //a[@class="series-item-logo"]/img/@alt
+      Image: //div[@class="player-thumb"]/img/@src
```
Comment (peolic) [inline scrapers/trafficpimps.yml:27]: I found this, does it work without CDP?
```suggestion
      Image: //div[@class="player-thumb"]/img/@src0_1
```

Comment (SpedNSFW) [reply scrapers/trafficpimps.yml:27]: No. Tried both.

Comment (peolic) [reply scrapers/trafficpimps.yml:27]: Sorry! Stupid typo (missed the `x` at the end). Also made sure it worked in Stash this time.
```suggestion
      Image: //div[@class="player-thumb"]/img/@src0_1x
```

Comment (SpedNSFW) [reply scrapers/trafficpimps.yml:27]: I had tried both `@src` and `@src0_1x` in Stash and neither worked until I added the CDP. See image below of my returned results using `@src0_1x`
image

**Pull request merged**